components/prism-solidity.js

```diff
@@ -0,0 +1,37 @@
+Prism.languages.solidity = Prism.languages.extend('javascript', {
+	'keyword': [
+		/\b(?:assert|break|constant|continue|do|else|emit|external|for|function|if|import|inherited|internal|is|memory|new|payable|private|public|pure|require|returns?|revert|selfdestruct|storage|suicide|this|throw|var|view|while|_)\b/,
+		{
+			pattern: /\b(?:contract|interface|library|using|struct|function|modifier)\s+[A-Za-z_]\w*(?:\s+is\s+(?:[A-Za-z_][,\s]*)*)?\b/,
+			inside: {
+				'variable': {
+					pattern: /\b(contract|interface|library|using|struct|function|modifier)\s+[A-Za-z_]\w*(?:\s+is\s+(?:[A-Za-z_][,\s]*)*)?\b/,
```

Comment:
Please move the `\s+` inside the lookbehind. Tokens should contain leading or trailing spaces.